### ISSUE TITLE
Retrieve current rotation

### ISSUE THREAD

**Issue by amenadiel** [godswearhats/jquery-ui-rotatable#11]
After rotating an object I need to retrieve the current angle so I can persist it in my backend.

At this moment, I'm doing this

```js
jQuery(div).rotatable({
		stop: function (element) {
			var rotation = jQuery(element.target).data('ui-rotatable').elementCurrentAngle;
			console.log('Rotatable stopped', rotation);
		}
	});
```

it works fine, but somehow it feels hacky. Is there a canonical way to get the widget attributes in a given moment?

**Comment by godswearhats** [comment]
Did anyone on this thread end up creating code to fix the problem?

**Comment by yaramayer** [comment]
@godswearhats
Speaking for myself, I worked with the value of elementStopAngle for a couple months and there was never an issue with the output. Judging from the code, that means that elementCurrentAngle should also be correct.
Regarding access, I only used jQuery('someelement').data('rotatable').elementCurrentAngle for testing purposes (it always worked) - I hooked into the stop events in my model and there used the ui.angle.x param.
Because of that, I didn't feel the need to change or add to the code. But maybe I missed the point/problem entirely.
